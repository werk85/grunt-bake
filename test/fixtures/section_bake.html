<!DOCTYPE html>
<html>
	<head></head>
	<body>
		<!--(bake includes/second.html _section="about")-->
		<!--(bake includes/second.html _section="home")-->
		<!--(bake includes/second.html _section="books.theDarkTower")-->

		<!--(bake-start _section="home")-->
		<span>{{title}}</span>
		<!--(bake-end)-->
	</body>
</html>