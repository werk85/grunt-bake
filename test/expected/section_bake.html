<!DOCTYPE html>
<html>
	<head></head>
	<body>
		<span>About</span>
		<span>Home</span>
		<span>The Dark Tower</span>

		<span>Home</span>
	</body>
</html>